.github/workflows/tests.yml

```diff
@@ -34,6 +34,24 @@ jobs:
         run: |
           cargo --version
           rustc --version
+
+      # FIXME: remove, trying out oudside pages/deploy
```

Comment:
I tried out https://github.com/taiki-e/install-action/tree/v2/ for cargo-mutants, and it looks like it might be a good way to get mdbook-linkcheck too?